src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,53 +1,138 @@
 package core.basesyntax;
 
+import java.util.NoSuchElementException;
+
 /**
  * <p>Реалізувати свій ArrayList який імплементує інтерфейс List. Дотриматися основних вимог щодо
  * реалізації ArrayList (default capacity, newCapacity...)</p>
  */
 public class ArrayList<T> implements List<T> {
+    private static final int DEFAULT_ARRAY_CAPASITY = 10;
+    private Object[] elementData;
+    private int arrayCapasity;
```

Comment:
you don't need separate variable for capacity, cause your capacity is always elementData.length